README.md

```diff
@@ -67,6 +67,48 @@ Complete the following steps before you deploy the Secure Landing Zone module.
 
     Select **All Identity and Access enabled services** when you assign access to the group.
 
+### Steps to prevent Destroy and recreation of cluster.
+1. Prior to `terraform plan` and `terraform apply`. Edit `moved_config.tf`
+2. Add the following lines to the terraform file.
```

Comment:
which file, `/patterns/roks/main.tf`?

Reply:
It could depend, it can be the /patterns/roks/main.tf or maybe the /examples/one-vsi-one-roks/main.tf